packages/p2p/src/components/my-profile/payment-methods/add-payment-method/add-payment-method.scss

```diff
@@ -52,6 +63,12 @@
 
         &__payment-method-field {
             margin: 3.8rem 0;
+
+            @include mobile {
+                &:last-of-type {
+                    margin-bottom: 10rem;
+                }
+            }
```

Comment:
Added bottom margin to offset the last form field in fixed footer:
<img width="506" alt="Screenshot 2022-05-24 at 2 49 31 PM" src="https://user-images.githubusercontent.com/103016120/171142653-81f8f08a-31e6-462a-870c-9c9f9e9da621.png">